doorknob/tests/test_foyer.py

```diff
@@ -2,6 +2,10 @@
 import unittest
 
 from doorknob.foyer import Scene
+<<<<<<< HEAD
+=======
+
+>>>>>>> 7947f58b3010bd6e94e4351e2d33a23f81608875
```

Comment:
@uyennguyen16900 please delete lines 5, 6, and 8. The git diff markers need to be removed, as they will cause the tests to fail.